components/UnifiedDocFeed/FeedMenu/FeedMenuTab.tsx

```diff
@@ -0,0 +1,130 @@
+import { tagFilters } from "../constants/UnifiedDocFilters";
+import FeedMenuTagDropdown from "./FeedMenuTagDropdown";
+import { SelectedUrlFilters } from "../utils/getSelectedUrlFilters";
+import { css, StyleSheet } from "aphrodite";
+import icons from "~/config/themes/icons";
+import colors from "~/config/themes/colors";
+import { breakpoints } from "~/config/themes/screen";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { buildTypeFilterUrl } from "../utils/buildTypeFilterUrl";
+
+
+type Args = {
+  selectedFilters: SelectedUrlFilters;
+  tabObj: any;
+  handleOpenTagsMenu: Function;
+  handleFilterSelect: Function;
+  isTagsMenuOpen: boolean;
+};
+
+const FeedMenuTab = ({
+  selectedFilters,
+  tabObj,
+  handleOpenTagsMenu,
+  handleFilterSelect,
+  isTagsMenuOpen,
+}: Args) => {
+  const router = useRouter();
+  const isSelected = tabObj.value === selectedFilters.type;
+  const nestedOptions = tagFilters.filter((sub) =>
+    sub.availableFor.includes(tabObj.value)
+  );
```

Comment:
Can we revisit this approach here? 
There's really no need for a rendering / child component to have been passed down a full tabObj to determine whether or not it's been selected. I think boolean variable might be sufficient in this case.

Reply:
Yeah, that's fair. I just updated.